### PR TITLE
Mdd 21.1 date picker

### PR DESCRIPTION
**Pull request by mddifilippo89**

### CODE REVIEW

**Comment by mddifilippo89** [comment]
@HUSSAR-mtrela  

I got latest on vnext and my branch is up to date, with no merge conflicts

File: doc/en/components/charts/features/chart-tooltips.md [hunk 1]
```diff
@@ -48,7 +50,7 @@ This example shows how to create custom tooltips for all series in $PlatformShor
 <div class="divider--half"></div>
 
 
-### Custom Tooltips in Data Chart
+## Custom Tooltips in Data Chart
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/features/chart-tooltips.md:53]: get latest and resolve conflict

File: doc/en/components/editors/date-picker.md [hunk 2]
```diff
@@ -57,9 +57,26 @@ The $PlatformShort$ Date Picker editor component consists of three default parts
...
-
-- Clear Button: The clear button is directly to the left of the calendar button, visualized by an X. When clicked, it will clear the `Value` of the Date Picker. The clear button's visibility can be toggled on and off by setting the `ShowClearButton` property of the Date Picker.
+- Calendar: Clicking the calendar button will open the calendar drop-down of the Date Picker. The calendar will open to show the month of the currently selected `Value`, if there is one. If a `Value` is not specified, the calendar will show the month corresponding to the current date. The month at the top can be clicked to show a list of months, and the same applies for the year. There are also arrow buttons to the left and right of the month and year visualization that will increment and decrement the shown month chronologically.
+
```
Comment (HUSSAR-mtrela) [inline doc/en/components/editors/date-picker.md:61]: missing a sample showing styling features

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]
change the first paragraph to:
The $ProductName$ DatePicker allows ...

**Comment by HUSSAR-mtrela** [comment]
@mddifilippo89 resolve conflicts

**Pull request merged**